extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -13,6 +14,7 @@ define('ext.wikia.adEngine.template.porvata', [
 	require.optional('ext.wikia.adEngine.mobile.mercuryListener')
 ], function (
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 12 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)